### DISCUSSION TITLE
Allow SN user accounts to be merged

### DISCUSSION THREAD

**Discussion by rleed** [stackernews/stacker.news#1879] [Ideas]
**Is your feature request related to a problem? Please describe.**
Sometimes I login to SN using a different method associated with the same identity. For example, I might normally login with GitHub, then one day I login with my e-mail address instead. This results in a new account, which would be immediately recognizable by the variation of the username and the zero balance.

**Describe the solution you'd like**
At that point, I would like to be able to merge the new account with my existing account (just by re-logging in with the old account's creds).

A pie-in-the-sky general solution could be a comprehensive merge of two arbitrary accounts. Not sure how practical it would be or how many people would use it... perhaps if entities want to anonymously "test" a new brand/personality before claiming ownership of it as part of their main identity, or for whatever reason they just have multiple active accounts and want to simplify and combine any of them.

**Describe alternatives you've considered**
- Delete the new account, login to the old account, manually set the additional auth method.
- Abandon the new account, login to the old account.

**Additional context**
#164

**Comment by huumn** [comment]
Nice write up! Curious to see what others think
